### PR TITLE
Refs #13431 - Apache changes for pulp 2.8

### PR DESCRIPTION
**Pull request by parthaa**
Refs #13431 - Apache changes for pulp 2.8 Includes updates for pulp.conf

### CODE REVIEW

File: templates/pulp.conf.erb [hunk 1]
```diff
@@ -17,15 +17,23 @@ SSLInsecureRenegotiation on
...
 #WSGIRestrictStdout Off
 
 WSGISocketPrefix run/wsgi
-WSGIScriptAlias /pulp/api /srv/pulp/webservices.wsgi
-WSGIImportScript /srv/pulp/webservices.wsgi process-group=pulp application-group=pulp
+WSGIScriptAlias /pulp/api /usr/share/pulp/wsgi/webservices.wsgi
+WSGIImportScript /usr/share/pulp/wsgi/webservices.wsgi process-group=pulp application-group=pulp
+
+<Directory /usr/share/pulp/wsgi>
+<%- if scope.function_versioncmp([@apache_version, '2.4']) >= 0 -%>
```
Comment (ehelms) [inline templates/pulp.conf.erb]: You will need to set this up -- for example: https://github.com/Katello/puppet-katello/blob/c12447f1c01ba837d5ddbe1aeb908e6d18e6ab57/manifests/config.pp#L4

File: templates/pulp.conf.erb [hunk 2]
```diff
@@ -38,9 +46,25 @@ WSGIImportScript /srv/pulp/webservices.wsgi process-group=pulp application-group
...
+    SSLRequireSSL
+    Options +Indexes
+<%- if scope.function_versioncmp([@apache_version, '2.4']) >= 0 -%>
+     Require all granted
+<% else -%>
+     Allow from all
+<% end -%>
+</Location>
+
 <%- if scope.lookupvar('operatingsystem') == 'Fedora' && scope.lookupvar('operatingsystemrelease') == '18' -%>
```
Comment (ehelms) [inline templates/pulp.conf.erb]: Feel free to ditch lines 61-73

**Comment by parthaa** [comment]
@ehelms updated

**Comment by ehelms** [comment]
ACK - thanks @parthaa !

**Pull request merged**